test/file.ts

```diff
@@ -3526,6 +3548,7 @@ describe('File', () => {
         };
         file.move('new-filename', (err: Error) => {
           assert.strictEqual(err, error);
+          assert.ok(err.message.includes('delete'));
```

Comment:
Could you elaborate this check to include the whole prefix string and original error message?

Reply:
Thanks, done here and by fail on copy above